contrib/golang/filters/http/source/go/pkg/api/api.h

```diff
@@ -6,6 +6,13 @@
 extern "C" {
 #endif
 
+// This describes the return value of a C Api from Go.
```

Comment:
Maybe it would be better to use enum type, and then use `CAPIStatus` instead of int type.
```
enum CAPIStatus {
……
}
```